hbase-server/src/main/java/org/apache/hadoop/hbase/coprocessor/MetaTableMetrics.java

```diff
@@ -203,41 +203,37 @@ private String opWithClientMeterName(Object op) {
         return "";
       }
       MetaTableOps ops = opsNameMap.get(op.getClass());
-      String opWithClientMeterName = "";
+      if (ops == null) {
```

Comment:
The changes in this file are all unrelated.

Reply:
These changes are part of backporting HBASE-26523 here: https://github.com/apache/hbase/commit/e53712ae993248594245363fc3e8e36022534d9a#diff-d10bebcd73c23f3ecb0565d7854f9a5659d33bf62baee31090ce6ef1ec224998R209-R211